Chat/src/app/utils/getToken.test.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Consider removing this file as we don't have testing enabled for this web app:
![image](https://user-images.githubusercontent.com/2684369/149030127-bc098eea-b9f4-4ddc-966a-579aafd762e3.png)

Reply:
Created ADO Item
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691384